src/document.c

```diff
@@ -248,17 +240,6 @@ void AddDocumentCtx_Finish(RSAddDocumentCtx *aCtx) {
 // How many bytes in a document to warrant it being tokenized in a separate thread
 #define SELF_EXEC_THRESHOLD 1024
 
-// LCOV_EXCL_START debug
-void Document_Dump(const Document *doc) {
```

Comment:
No, didn't feel like it is something we would need in a production environment by the end user.
And we shouldn't run it since we shouldn't get user data access.
And didn't see the need for it in tests.